e2e/tests/user/registration-ui.spec.ts

```diff
@@ -3,56 +3,19 @@ import SelectComponent from '../../app/aou-elements/select-component';
 import Textbox from '../../app/aou-elements/textbox';
 import CreateAccountPage, {FIELD_LABEL, INSTITUTION_ROLE_VALUE, INSTITUTION_VALUE} from '../../app/create-account-page';
 import GoogleLoginPage from '../../app/google-login';
-import PuppeteerLaunch from '../../driver/puppeteer-launch';
 
 const configs = require('../../resources/workbench-config');
 
-// set timeout globally per suite, not per test.
-jest.setTimeout(2 * 60 * 1000);
 
-describe.skip('User registration tests:', () => {
-
-  let browser;
-  let page;
-
-  beforeAll(async () => {
-    browser = await PuppeteerLaunch();
-  });
+describe('User registration tests:', () => {
 
   beforeEach(async () => {
-    page = await browser.newPage();
-    await page.setDefaultNavigationTimeout(60000);
+    await page.setUserAgent(configs.puppeteerUserAgent);
+    await page.setDefaultNavigationTimeout(120000);
   });
 
   afterEach(async () => {
-    await page.close();
-  });
-
-  afterAll(async () => {
-    await browser.close();
-  });
-
-
-  test('Entered invalid invitation key', async () => {
```

Comment:
test is no longer valid due to removal of invitation key during Create Account workflow.